docs/bigdata/README.md

```diff
@@ -0,0 +1,272 @@
+
+*   _fs.s3a.access.key=minio_ (Access Key to access MinIO instance, this is obtained after the deployment on k8s)
+*   _fs.s3a.secret.key=minio123_ (Secret Key to access MinIO instance, this is obtained after the deployment on k8s)
+*   _fs.s3a.endpoint=http://minio-address/_
+*   _fs.s3a.path.style.acces=true_
+
+Rest of the other optimization options are discussed here
+
+*   [https://hadoop.apache.org/docs/current/hadoop-aws/tools/hadoop-aws/index.html](https://hadoop.apache.org/docs/current/hadoop-aws/tools/hadoop-aws/index.html)
+*   [https://hadoop.apache.org/docs/r3.1.1/hadoop-aws/tools/hadoop-aws/committers.html](https://hadoop.apache.org/docs/r3.1.1/hadoop-aws/tools/hadoop-aws/committers.html)
+
+![alt_text](images/image5.png "image_tooltip")
+
+Once the config changes are applied, proceed to restart **Hadoop** services.
+
+![alt_text](images/image6.png "image_tooltip")
+
+### **3.2 Configure Spark2**
+
+Navigate to **Services** -> **Spark2** -> **CONFIGS** as shown below
+
+![alt_text](images/image7.png "image_tooltip")
+
+Navigate to “**Custom spark-defaults**” to configure MinIO parameters for `_s3a_` connector
+
+![alt_text](images/image8.png "image_tooltip")
+
+
+Add the following optimal entries for _spark-defaults.conf_ to configure Spark with **MinIO**.
+
+*   _spark.hadoop.fs.s3a.committer.magic.enabled=true_
+*   _spark.hadoop.fs.s3a.committer.name=magic_
+*   _spark.hadoop.fs.s3a.impl=org.apache.hadoop.fs.s3a.S3AFileSystem_
+*   _spark.hadoop.fs.s3a.path.style.access=true_
+*   _spark.hadoop.mapreduce.outputcommitter.factory.scheme.s3a=org.apache.hadoop.fs.s3a.commit.S3ACommitterFactory_
+
+![alt_text](images/image9.png "image_tooltip")
+
+Once the config changes are applied, proceed to restart **Spark** services.
+
+
+![alt_text](images/image10.png "image_tooltip")
+
+
+
+### **3.3 Configure Hive**
+
+Navigate to **Services** -> **Hive** -> **CONFIGS **-> **ADVANCED** as shown below
+
+
+![alt_text](images/image11.png "image_tooltip")
+
+
+Navigate to “**Custom hive-site**” to configure MinIO parameters for `_s3a_` connector
+
+
+![alt_text](images/image12.png "image_tooltip")
+
+
+Add the following optimal entries for `**_hive-site.xml`_** to configure Hive with **MinIO**.
+
+*   _hive.blobstore.use.blobstore.as.scratchdir=true_
+*   _hive.exec.input.listing.max.threads=50_
+*   _hive.load.dynamic.partitions.thread=25_
+*   _hive.metastore.fshandler.threads=50_
+*   _hive.mv.files.threads=40_
+*   _mapreduce.input.fileinputformat.list-status.num-threads=50_
+
+To know more information about these options please visit [https://www.cloudera.com/documentation/enterprise/5-11-x/topics/admin_hive_on_s3_tuning.html](https://www.cloudera.com/documentation/enterprise/5-11-x/topics/admin_hive_on_s3_tuning.html)
+
+
+![alt_text](images/image13.png "image_tooltip")
+
+
+Once the config changes are applied, proceed to restart all Hive services.
+
+
+![alt_text](images/image14.png "image_tooltip")
```

Comment:
Image shows Stop and Restart highlighted, we should only show the correct option